bin/runtime/src/lib.rs

```diff
@@ -334,13 +334,16 @@ parameter_types! {
     // We allow 10kB keys, proofs and public inputs. This is a 100% blind guess.
     pub const MaximumVerificationKeyLength: u32 = 10_000;
     pub const MaximumDataLength: u32 = 10_000;
+    pub const VerificationKeyDepositAmount: u128 = 1_000_000_000;
```

Comment:
```suggestion
    pub const VerificationKeyDepositPerByte: u128 = 1_000_000_000;
```

Reply:
and we can use either `LEGACY_DEPOSIT_PER_BYTE`, `CONTRACT_DEPOSIT_PER_BYTE` or `MILLI_AZERO`

Reply:
done